[2022-05-19 03:23:36,340] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Teste_ETL_Raízen.captura_conta_dados manual__2022-05-19T03:19:54.300030+00:00 [queued]>
[2022-05-19 03:23:36,352] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Teste_ETL_Raízen.captura_conta_dados manual__2022-05-19T03:19:54.300030+00:00 [queued]>
[2022-05-19 03:23:36,353] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-05-19 03:23:36,353] {taskinstance.py:1357} INFO - Starting attempt 3 of 3
[2022-05-19 03:23:36,354] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-05-19 03:23:36,372] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): captura_conta_dados> on 2022-05-19 03:19:54.300030+00:00
[2022-05-19 03:23:36,378] {standard_task_runner.py:52} INFO - Started process 3323 to run task
[2022-05-19 03:23:36,381] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Teste_ETL_Raízen', 'captura_conta_dados', 'manual__2022-05-19T03:19:54.300030+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/ETL_Raizen.py', '--cfg-path', '/tmp/tmpn9fhbkmy', '--error-file', '/tmp/tmpraivfsg5']
[2022-05-19 03:23:36,383] {standard_task_runner.py:80} INFO - Job 8: Subtask captura_conta_dados
[2022-05-19 03:23:36,451] {task_command.py:369} INFO - Running <TaskInstance: Teste_ETL_Raízen.captura_conta_dados manual__2022-05-19T03:19:54.300030+00:00 [running]> on host c8caf9da75ac
[2022-05-19 03:23:36,532] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Teste_ETL_Raízen
AIRFLOW_CTX_TASK_ID=captura_conta_dados
AIRFLOW_CTX_EXECUTION_DATE=2022-05-19T03:19:54.300030+00:00
AIRFLOW_CTX_TRY_NUMBER=3
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-05-19T03:19:54.300030+00:00
[2022-05-19 03:23:39,574] {python.py:173} INFO - Done. Returned value was: 807
[2022-05-19 03:23:39,606] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Teste_ETL_Raízen, task_id=captura_conta_dados, execution_date=20220519T031954, start_date=20220519T032336, end_date=20220519T032339
[2022-05-19 03:23:39,646] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-05-19 03:23:39,691] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
